### PR TITLE
Fix typos in openstack-cli tutorials

### PR DESCRIPTION
**Pull request by bpetermannS11**
Fixes some typos in English and German openstack-cli.md

### CODE REVIEW

File: english/docs/tutorials/openstack-cli.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ for plugin in openstack cinder glance heat neutron nova ; do sudo pip install py
 **Python**
 
 To be able to use the OpenStack Client on Windows we first need [Python 2.7](https://www.python.org/downloads/release/python-2712/).
-After the installation is finished, we open our Command Promt and ensure that we're in the following directory: `C:\Python27\Scripts`
+After the installation is finished, we open our Command Prompt and ensure that we're in the following directory: `C:\Python27\Scripts`
```
Comment (stefreak) [inline english/docs/tutorials/openstack-cli.md]: nouns are uncapitalized in english; correct would be  `command prompt`

File: german/docs/tutorials/openstack-cli.md [hunk 2]
```diff
@@ -4,26 +4,26 @@
 
 ## Übersicht
 
-Dieses Tutorial dient als Anleitung für die Installation des Openstack CLI (Command Line Interfacs). Mit dem Openstack CLI kannst du deine Stacks administrieren und überwachen.
+Dieses Tutorial dient als Anleitung für die Installation des Openstack CLI (Command Line Interface). Mit dem Openstack CLI kannst du deine Stacks administrieren und überwachen.
```
Comment (stefreak) [inline german/docs/tutorials/openstack-cli.md]: `command line interface`

File: german/docs/tutorials/openstack-cli.md [hunk 3]
```diff
@@ -4,26 +4,26 @@
...
 
 ## Ziel
 
-* Openstack-Commandline Client in einer aktuellen Version und benötigte Plugins installieren
+* Openstack Command Line Client in einer aktuellen Version und benötigte Plugins installieren
```
Comment (stefreak) [inline german/docs/tutorials/openstack-cli.md]: Command-Line

**Comment by stefreak** [comment]
👍

File: english/docs/tutorials/openstack-cli.md (hunk 1 above)
Comment (bpetermannS11) [reply english/docs/tutorials/openstack-cli.md]: fixed

File: german/docs/tutorials/openstack-cli.md (hunk 2 above)
Comment (bpetermannS11) [reply german/docs/tutorials/openstack-cli.md]: changed

File: german/docs/tutorials/openstack-cli.md (hunk 3 above)
Comment (bpetermannS11) [reply german/docs/tutorials/openstack-cli.md]: changed

**Comment by dschwabeS11** [comment]
Thanks for fixing those.

**Pull request merged**